CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
 - Add --version option
 - Add -v|--verbose option
 - Add ASCII art logo
+- Add option `--stop-on-failure`
```

Comment:
Can you move this line to the current `Unreleased` section (the one above) 👆

Reply:
Yes, for sure